web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
+            <li><a href="{{ pathPrefix }}/config">Config</a></li>
```

Comment:
That's merely one debugging task, there's many others such as debugging rules, alerts, storage, version, flags etc.
There are too many of these to put them all on the navbar, and targets isn't obviously more important than the rest (from experience it's actually one of the less important ones, as once it works it tends to keep on working forever).